test/library/standard/Sort/performance/sort-performance-explorer.chpl

```diff
@@ -2,9 +2,11 @@
 // for differing algorithms, input data distributions, and input sizes.
 
 use Sort;
+use TwoArraySampleSort;
 use Random;
 use Time;
 use CTypes;
+import MergeSort;
```

Comment:
Is there a reason why MergeSort is `import`ed but the others are `use`d?
It looks like all of the sort calls are qualified with the module name so they could all be done the same way?

Reply:
I added `MergeSort` as an import because it was qualified, but was not consistent when I added `TwoArraySampleSort` in a later commit. I would rather just make it `import MergeSort, TwoArraySampleSort`.

Reply:
I did this in https://github.com/chapel-lang/chapel/pull/25703/commits/f92ff3ff4a724fd5aaaabc1bf59869469ef7fee7, but I can also make it all `use` if you prefer

Reply:
This works